.travis.yml

```diff
@@ -31,14 +31,6 @@ matrix:
       # its size category trait on rotation.
       # 10.3 was chosen to represent iOS 10.
       env: DESTINATION="platform=iOS Simulator,name=iPhone 7 Plus,OS=10.3.1" SDK="iphonesimulator10.3.1"
```

Comment:
Consider re-adding an xcode 9 - iOS 9 simulator for coverage.

Reply:
Done.